rabbit-escape-render/src/rabbitescape/render/gameloop/GeneralPhysics.java

```diff
@@ -18,15 +19,21 @@
     public static class WorldModifier
     {
         private final World world;
+        public final SolutionRecorder solutionRecorder;
 
-        public WorldModifier( World world )
+        public WorldModifier( World world, SolutionRecorder solutionRecorder )
         {
             this.world = world;
+            this.solutionRecorder = solutionRecorder;
         }
 
         public synchronized void step()
         {
             world.step();
+            if ( null != solutionRecorder )
+            {
```

Comment:
I am soon going ya make it possible/mandatory to compile Android with each build.